app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -4,7 +4,7 @@ class PermissionTemplateForm
       include HydraEditor::Form
       self.model_class = PermissionTemplate
       self.terms = []
-      delegate :access_grants, :access_grants_attributes=, :visibility, to: :model
+      delegate :access_grants, :access_grants_attributes=, :visibility, :workflow_name, to: :model
```

Comment:
Removed. For some reason I thought I'd tested and it didn't work without it, but I'm guessing that was related to that permissions_template issue.